stable/chronograf/values.yaml

```diff
@@ -12,7 +12,7 @@ service:
 ## Persist data to a persitent volume
 persistence:
   enabled: false
-  storageClass: generic
+  storageClass: ""
```

Comment:
I had thought I switched it all to be commented out. I'm going to upload a couple more commits to this with that and a rebasing against master shortly, and have the commands I need all setup into a ugly one-liner to do the split into N different branches at least.